pams/sim/idl/g2t_tpc_hit.idl

```diff
@@ -16,4 +16,7 @@
        long      np;         /* no. of primary interactions per hit */
```

Comment:
I don't see any technical issues with this change to the g2t_tpc_hit structure.  But there is an architectural one.
The g2t tables define (or at least express) the MC event data model.  The data members being added are not
scored by geant simulation, but by a later stage of event processing / digitization.  This is a mixing of concerns.

I would prefer that a separate data structure within TpcRS handle this additional information (along with the np, ne and adc fields).   This can be addressed in a future PR.

Reply:
FilePath structure I do need to use exactly the same gain files as Tonko uses during data taking. It is supposed to be used  as Cint files only. Really it is provide a time stamp to Tonko's gain files.

Yes, I agree that I miss modifications in StMcEvent. The modified structure is used in StTpcMcAnalysisMaker to tune up StTpcRSMaker.
I will add to this PR modification to StMcEvent